### PR TITLE
Fix language errors in extensions

### PR DESCRIPTION
**Pull request by themr0c**
Fix vale errors in extensions.

```
vale --minAlertLevel=error modules/extensions/

 modules/extensions/partials/assembly_openshift-connector-overview.adoc
 14:280  error  Consider using 'rather than'    IBM.Terms 
                instead of 'Instead of'                   


 modules/extensions/partials/proc_creating-components-with-openshift-connector-in-che.adoc
 8:170  error  Did you really mean             Vale.Spelling 
               'deployables'?                                


 modules/extensions/partials/proc_creating-a-telemetry-plugin.adoc
 121:60   error  Avoid using 'our'.              Vale.Avoid    
 124:19   error  Use 'Java' instead of 'java'.   Vale.Terms    
 130:20   error  Use 'Java' instead of 'java'.   Vale.Terms    
 140:19   error  Use 'Java' instead of 'java'.   Vale.Terms    
 150:139  error  Avoid using 'our'.              Vale.Avoid    
 152:19   error  Use 'Java' instead of 'java'.   Vale.Terms    
 158:70   error  Did you really mean             Vale.Spelling 
                 'debounces'?                                  
 164:19   error  Use 'Java' instead of 'java'.   Vale.Terms    
 174:19   error  Use 'Java' instead of 'java'.   Vale.Terms    
 184:19   error  Use 'Java' instead of 'java'.   Vale.Terms    
 206:89   error  Avoid using 'we'.               Vale.Avoid    
 208:10   error  Use 'ConfigMap' instead of      Vale.Terms    
                 'configMap'.                                  
 215:102  error  Use 'ConfigMap' instead of      Vale.Terms    
                 'configMap'.                                  


 modules/extensions/partials/assembly_eclipse-che4z.adoc
 50:11   error  Avoid using 'we'.  Vale.Avoid 
 73:11   error  Avoid using 'we'.  Vale.Avoid 
 93:11   error  Avoid using 'we'.  Vale.Avoid 
 108:11  error  Avoid using 'we'.  Vale.Avoid 
 122:11  error  Avoid using 'we'.  Vale.Avoid 


 modules/extensions/partials/ref_the-woopra-telemetry-plugin.adoc
 5:509  error  Use 'Woopra' instead of         Vale.Terms 
               'woopra'.                                  


 modules/extensions/partials/proc_installing-openshift-connector-in-che.adoc
 22:22  error  Use 'OpenShift' instead of      Vale.Terms 
               'openshift'.                               


 modules/extensions/partials/con_features-of-openshift-connector.adoc
 37:47  error  Use 'Pods' instead of 'pods'.  Vale.Terms 


 modules/extensions/partials/proc_java-lombok.adoc
 1:16  error  Did you really mean 'lombok'?  Vale.Spelling 

✖ 24 errors, 0 warnings and 0 suggestions in 23 files.

```

### PR Checklist

As the author of this Pull Request I made sure that:

- [ x] `vale` has been run successfully against the PR branch
- [x] Link checker has been run successfully against the PR branch
- [x] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [x] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [x] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [x] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Review by yhontyk — APPROVED** [review]

File: modules/extensions/partials/proc_creating-a-telemetry-plugin.adoc [hunk 1]
```diff
@@ -198,27 +218,31 @@ See link:https://quarkus.io/guides/building-native-image#using-a-multi-stage-doc
...
 .meta.yaml
+====
 [source,yaml]
 ----
 include::example$telemetry/sample_meta.yaml[]
 ----
+====
 
-In most cases, a user would deploy this file to a corporate web server. For this guide, we create an Apache web server on OpenShift and host the plug-in there.
+In most cases, a user would deploy this file to a corporate web server. For this guide, create an Apache web server on OpenShift and host the plug-in there.
```
Comment (ericwill) [inline modules/extensions/partials/proc_creating-a-telemetry-plugin.adoc]: Removing the `we` in this sentence changes its meaning. I'd put it back in.

Comment (themr0c) [reply modules/extensions/partials/proc_creating-a-telemetry-plugin.adoc]: `we` is one of words we reject.

Comment (themr0c) [reply modules/extensions/partials/proc_creating-a-telemetry-plugin.adoc]: You got a point: the complete procedure should be reviewed for modularity and language.
I added all errors reported by vale in the description of the PR to make it clear

Comment (yhontyk) [reply modules/extensions/partials/proc_creating-a-telemetry-plugin.adoc]: Maybe reword the whole sentence? 
"This guide demonstrates how to create an Apache..."
or
"This section guides you..."
?

Comment (themr0c) [inline modules/extensions/partials/proc_creating-a-telemetry-plugin.adoc]:
```suggestion
In most cases, a user would deploy this file to a corporate web server. This guide demonstrates how to create an Apache web server on OpenShift and host the plug-in there.
```

Comment (themr0c) [reply modules/extensions/partials/proc_creating-a-telemetry-plugin.adoc]: "This guide demonstrates how to create an Apache..." let's go.

Followup: https://github.com/eclipse/che/issues/19567

**Review by ericwill — APPROVED** [review]
